src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+$count: (0, 1, 2, 3, 4, 5);
+
+.stars {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+
+  &__star {
+    background: center url(../images/star.svg) no-repeat;
+    margin-left: 4px;
+    height: 16px;
+    width: 16px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  @each $amount in $count {
```

Comment:
You can just just `for` - https://sass-lang.com/documentation/at-rules/control/for/